### PR TITLE
RDM-12701: Remove Embedded Postgres

### PR DESCRIPTION
**Pull request by sullivangareth**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RDM-12701

### Change description ###

Removing the embedded postgres from CCD-Data-Store

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/test/java/uk/gov/hmcts/ccd/endpoint/std/CallbackTest.java [hunk 1]
```diff
@@ -1405,7 +1403,6 @@ public void shouldReturn404WhenPostCreateEventWithInvalidEventTokenForCitizen()
 
     @Test
     @Sql(executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD, scripts = {"classpath:sql/insert_callback_cases.sql"})
-    @Transactional
```
Comment (smathangi) [inline src/test/java/uk/gov/hmcts/ccd/endpoint/std/CallbackTest.java:1408]: are we sure @Transactional is not required here?

**Review by smathangi — COMMENTED** [review]
Just one comment otherwise LGTM.

File: src/test/java/uk/gov/hmcts/ccd/endpoint/std/CallbackTest.java (hunk 1 above)
Comment (sullivangareth) [reply src/test/java/uk/gov/hmcts/ccd/endpoint/std/CallbackTest.java:1408]: Not completely sure, but with it left in the test consistently fails with the following postgres error

`ERROR: canceling statement due to user request`

I have tried playing with timeouts, but without much luck

Comment (smathangi) [reply src/test/java/uk/gov/hmcts/ccd/endpoint/std/CallbackTest.java:1408]: Ok, if the test passes repeatedly without that annotation then we're good.

**Review by smathangi — APPROVED** [review]

File: src/test/resources/integration_tests.properties [hunk 2]
```diff
@@ -2,3 +2,6 @@
 idam.api.url=http://localhost:${wiremock.server.port}/idam
 
 idam.s2s-auth.url=http://localhost:${wiremock.server.port}/s2s
+
```
Comment (mattnayler) [inline src/test/resources/integration_tests.properties:5]: Is this file needed now that `src/test/java/uk/gov/hmcts/ccd/integrations/IntegrationTest.java` has been removed?

**Review by mattnayler — APPROVED** [review]
Just one question otherwise LGTM 👍 .

File: src/test/resources/integration_tests.properties (hunk 2 above)
Comment (sullivangareth) [reply src/test/resources/integration_tests.properties:5]: Good point, will check and remove accordingly

Comment (sullivangareth) [reply src/test/resources/integration_tests.properties]: Fixed in b357d60

File: build.gradle [hunk 3]
```diff
@@ -214,13 +214,13 @@ dependencies {
 
     // FIXME 0.6 doesn't support jsonb; 0.7 doesn't work on Windows
 //    runtime group: 'com.impossibl.pgjdbc-ng', name: 'pgjdbc-ng', version: '0.6'
-    runtime group: 'org.postgresql', name: 'postgresql', version: '42.2.16'
+    runtime group: 'org.postgresql', name: 'postgresql', version: '42.2.23'
```
Comment (smathangi) [inline build.gradle:217]: if this bump is not required for test container, could you remove please. We can do that in another PR. So that we can release this PR as test only changes.

Comment (sullivangareth) [reply build.gradle:217]: 👍

Comment (sullivangareth) [reply build.gradle]: Fixed in 640c299
